manifests/windows.pp

```diff
@@ -16,7 +17,13 @@
   Enum['present', 'absent'] $ensure = 'present',
 ) inherits datadog_agent::params {
 
-  $msi_full_path = "${msi_location}/datadog-agent-6-${agent_version}.amd64.msi"
+  $msi_full_path = "${msi_location}/datadog-agent-${agent_major_version}-${agent_version}.amd64.msi"
+
+  if ($agent_repo_uri != undef) {
+    $baseurl = $agent_repo_uri
+  } else {
+    $baseurl = 'https://s3.amazonaws.com/ddagent-windows-stable/'
+  }
 
   if $agent_version == 'latest' {
     $msi_source = "${baseurl}datadog-agent-6-latest.amd64.msi"
```

Comment:
This should be set according to the major version: here, whatever the major version set, it will install 6-latest.

Reply:
Fixed 👍